compliance_checker/tests/test_cli.py

```diff
@@ -92,6 +95,41 @@ def test_single_json_output(self):
             r = json.load(f)
             assert 'cf' in r
 
+
+    def test_list_checks(self):
+        """
+        Tests listing of both old-style, deprecated checkers using .name
+        attributes, and newer ones which use ._cc_spec and _cc_spec_version
+        attributes
+        """
+
+        # hack: use argparse.Namespace to mock checker object with attributes
+        # since SimpleNamespace is Python 3.3+ only
```

Comment:
are we still supporting <3.3?!?

Reply:
As Python 2.7 isn't deprecated yet and much of the code supports Python 2.7, yes.  Handling of various Python 2.7 quirks/differences, especially Unicode, has been a pain point.  Thus, I'd like to do away with backwards compatibility with Python 2.7 in the next major version, especially with the upcoming Python 2.7 EOL.  See https://github.com/ioos/compliance-checker/issues/557 , which I closed mainly to clean up some issues.

Reply:
sure that's fine.  maybe add this as a list item of pending things to clear out when 2.7 is yanked.